index.html

```diff
@@ -65,147 +66,158 @@ <h1 class="intro">I'm Ernest Glad to see you!</h1>
+           
         </div>
     </section>
 
     <!--About Section-->
     <section class="about">
-        <div class="about-me">
-            <h3 class="intro">About <br>Myself</h3>
-            <p class="main-paragraph">Hello, I'm a software developer! I can help you build a product, feature or website. 
-                Look through some of my work and experience! If you like what you see and have a project you need coded,
-                 don't hesitate to contact me.
-            </p>
-            <p class="connect">LET'S CONNECT</p>
-        </div>
-        
-        <div class="headline-section">
-            <ul>
-                <li>
-                    <a href="#" class="social twiter">
-                        <img src="./images/twiter.png" alt="Twitter"/>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="social linkendIn">
-                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="social twiter">
-                        <img src="./images/mail.png" alt="Email"/>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="social github">
-                        <img src="./images/Vector.png" alt="GitHub"/>
-                    </a>
-                </li>
-                <li>
-                    <a href="#" class="social peace">
-                        <img src="./images/fingers.png" alt="Peace"/>
-                    </a>
-                </li>
-            </ul>
+        <div class="about-wrapper">
+            <div class="about-me">
+                <h3 class="intro">About <br>Myself</h3>
+                <p class="main-paragraph">Hello, I'm a software developer! I can help you build a product, feature or website. 
+                    Look through some of my work and experience! If you like what you see and have a project you need coded,
+                     don't hesitate to contact me.
+                </p>
+                <p class="connect">LET'S CONNECT</p>
+            </div>
+            
+            <div class="headline-section">
+                <ul>
+                    <li>
+                        <a href="#" class="social twiter">
+                            <img src="./images/twiter.png" alt="Twitter"/>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="social linkendIn">
+                            <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="social twiter">
+                            <img src="./images/mail.png" alt="Email"/>
+                        </a>
+                    </li>
+                    <li>
```

Comment:
- [ ] There are 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. Kindly implement it